querybook/webapp/components/DataDocViewersList/DataDocViewersList.tsx

```diff
@@ -54,6 +59,15 @@ export const DataDocViewersList: React.FunctionComponent<IDataDocViewersListProp
     updateDataDocOwner,
     rejectDataDocAccessRequest,
 }) => {
+    const dispatch: Dispatch = useDispatch();
+    const handleDataDocAccessRequest = React.useCallback(() => {
+        dispatch(addDataDocAccessRequest(dataDoc.id));
+    }, [dataDoc.id]);
+
+    const { isEditor } = useSelector((state: IStoreState) => ({
```

Comment:
const isEditor = useSelector((state: IStoreState) => canCurrentUserEditSelector(state, dataDoc.id));

Reply:
i did it this way so you could do this: // TODO: make this component use React-Redux directly
more easily later

Reply:
i did it this way so you could do this: // TODO: make this component use React-Redux directly
more easily later